### PR TITLE
Gblessylva

### PR DESCRIPTION
**Pull request by gblessylva**
## Description
Created the Login pages for Investors and SMEs as well as the Registration pages. Also Made it responsive.

Fixed issue #16 

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ x] I have added tests that prove my fix is effective and that this feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/styles/login.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+    height: 100%;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+.landing>.login-sidbar{
```
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
+    margin:0;
+}
+html{
```
Comment (oyekanmiayo) [inline src/styles/login.css:6]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

**Comment by oyekanmiayo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 81
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130&bid=18211889)

File: src/styles/login.css [hunk 3]
```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
```
Comment (oyekanmiayo) [inline src/styles/login.css:3]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/routes/beforeAuth.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Homepage from '../views/homepage/homepage';
+import LoginLanding from '../views/Login/LoginLanding.jsx';
+import InvestorsSignup from '../views/investors/SignupLanding'
+import InvestorsLogin from '../views/investors/InvestorsLoginLanding'
```
Comment (oyekanmiayo) [inline src/components/routes/beforeAuth.js:6]: Codacy Issue found: [Missing file extension for "../views/investors/InvestorsLoginLanding"](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/views/signup/smeSignup.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  }
+];
+
+class SmeSignup extends Component {
+  render() {
+    return (
+        <div className="login-wrapper signup-wrapper" >
+          <form action="">
+            <div className='login-form-header'>
+              <h2 className='signup-sub'>{this.props.subHeader}</h2>
```
Comment (oyekanmiayo) [inline src/components/views/signup/smeSignup.jsx:27]: Codacy Issue found: ['subHeader' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    /* background-color: #fff2ff; */
+    height: 90vh;
+    
+    
+}
+.landing>.login-wrapper{
+    width:90%;
```
Comment (oyekanmiayo) [inline src/styles/login.css:44]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 7]
```diff
@@ -0,0 +1,296 @@
...
+    align-items: center;
+    flex-direction: row;
+    padding:2rem;
+    
+}
+.landing>.login-wrapper{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
```
Comment (oyekanmiayo) [inline src/styles/login.css:25]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+    width: 95%;
+    margin: 0 auto;
+    display: flex;
+    justify-content:center;
+    align-items: center;
+    flex-direction: row;
+    padding:2rem;
+    
+}
+.landing>.login-wrapper{
```
Comment (oyekanmiayo) [inline src/styles/login.css:21]: Codacy Issue found: [Expected single space after ">" (selector-combinator-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+    flex-direction: column;
+    text-align: center;
+    /* background-color: #fff2ff; */
+    height: 90vh;
+    
+    
+}
+.landing>.login-wrapper{
+    width:90%;
+    margin:0 auto;
```
Comment (oyekanmiayo) [inline src/styles/login.css:45]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/views/investors/InvestorsLoginLanding.jsx [hunk 10]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Login from './Login.jsx';
+import LoginSideBar from '../../common/InvestorsLoginSidebar';
```
Comment (oyekanmiayo) [inline src/components/views/investors/InvestorsLoginLanding.jsx:3]: Codacy Issue found: [Missing file extension for "../../common/InvestorsLoginSidebar"](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 11]
```diff
@@ -0,0 +1,296 @@
...
+*{
+    padding:0;
+    margin:0;
+}
+html{
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+    
+}
+.landing{
```
Comment (oyekanmiayo) [inline src/styles/login.css:11]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/views/investors/Login.jsx [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+];
+
+class Login extends Component {
+  render() {
+    return (
+        <div className="login-wrapper">
+          <form action="">
+            <div className='login-form-header'>
+              <h1>{this.props.header}</h1>
+              <h2>{this.props.subHeader}</h2>
```
Comment (oyekanmiayo) [inline src/components/views/investors/Login.jsx:22]: Codacy Issue found: ['subHeader' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/LoginSidebar.jsx [hunk 13]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import step1img from '../../images/bgsvg.svg'
+
+class LoginSideBar extends Component {
+  render() {
+    return (
+            <div className='login-sidbar'>
+              <div className="sidebar-heading">
+                <h1>{this.props.heading} <span className='green-span'></span></h1>
+                <h2>{this.props.subHeading} </h2>
```
Comment (oyekanmiayo) [inline src/components/common/LoginSidebar.jsx:10]: Codacy Issue found: ['subHeading' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/views/investors/SignUp.jsx [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+  }
+];
+
+class Signup extends Component {
+  render() {
+    return (
+        <div className="login-wrapper signup-wrapper" >
+          <form action="">
+            <div className='login-form-header'>
+              <h2 className='signup-sub'>{this.props.subHeader}</h2>
```
Comment (oyekanmiayo) [inline src/components/views/investors/SignUp.jsx:27]: Codacy Issue found: ['subHeader' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 1 above)
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected single space before ">" (selector-combinator-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/Fields.jsx [hunk 15]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Field extends Component {
+  render() {
+    return (
+            <div className="form-group">
+                <label htmlFor={this.props.htmlFor} className='form-label'>{this.props.label}</label>
+                <input
+                    className="form-control"
+                    id={this.props.name}
```
Comment (oyekanmiayo) [inline src/components/common/Fields.jsx:10]: Codacy Issue found: ['name' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 1 above)
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected single space after ">" (selector-combinator-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 16]
```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
+    margin:0;
+}
+html{
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+    
+}
```
Comment (oyekanmiayo) [inline src/styles/login.css:10]: Codacy Issue found: [Unexpected empty line before closing brace (block-closing-brace-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 17]
```diff
@@ -0,0 +1,296 @@
...
+    height: 100%;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+.landing>.login-sidbar{
+    flex:2;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    /* background-color: #fff2ff; */
```
Comment (oyekanmiayo) [inline src/styles/login.css:38]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/views/investors/SignupLanding.jsx [hunk 18]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Signup from './SignUp.jsx';
+import LoginSideBar from '../../common/InvestorsLoginSidebar';
```
Comment (oyekanmiayo) [inline src/components/views/investors/SignupLanding.jsx:3]: Codacy Issue found: [Missing file extension for "../../common/InvestorsLoginSidebar"](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 19]
```diff
@@ -0,0 +1,296 @@
...
+html{
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+    
+}
+.landing{
+    width: 95%;
+    margin: 0 auto;
+    display: flex;
+    justify-content:center;
```
Comment (oyekanmiayo) [inline src/styles/login.css:15]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 1 above)
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 20]
```diff
@@ -0,0 +1,296 @@
...
+    /* background-color: #fff2ff; */
+    height: 90vh;
+    
+    
+}
+.landing>.login-wrapper{
+    width:90%;
+    margin:0 auto;
+}
+.landing>.login-wrapper>form>.login-form-header>h1{
```
Comment (oyekanmiayo) [inline src/styles/login.css:47]: Codacy Issue found: [Expected single space after ">" (selector-combinator-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/routes/beforeAuth.js [hunk 21]
```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Homepage from '../views/homepage/homepage';
+import LoginLanding from '../views/Login/LoginLanding.jsx';
+import InvestorsSignup from '../views/investors/SignupLanding'
```
Comment (oyekanmiayo) [inline src/components/routes/beforeAuth.js:5]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 22]
```diff
@@ -0,0 +1,296 @@
...
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+    height: 100%;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+.landing>.login-sidbar{
+    flex:2;
```
Comment (oyekanmiayo) [inline src/styles/login.css:33]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

**Review by gblessylva — COMMENTED** [review]
Done

**Comment by oyekanmiayo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 72
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130&bid=18211889)

File: src/styles/login.css (hunk 8 above)
Comment (oyekanmiayo) [inline src/styles/login.css:21]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 23]
```diff
@@ -0,0 +1,296 @@
...
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    /* background-color: #fff2ff; */
+    height: 90vh;
```
Comment (oyekanmiayo) [inline src/styles/login.css:39]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 24]
```diff
@@ -0,0 +1,296 @@
...
+    margin:0;
+}
+html{
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+    
+}
+.landing{
+    width: 95%;
+    margin: 0 auto;
```
Comment (oyekanmiayo) [inline src/styles/login.css:13]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 20 above)
Comment (oyekanmiayo) [inline src/styles/login.css:47]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 3 above)
Comment (oyekanmiayo) [inline src/styles/login.css:3]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 25]
```diff
@@ -0,0 +1,296 @@
...
+    }
+    80%{
+        content: 'Mature';
+    }
+    100%{
+        content: 'Connect';
+    }
+}
+.signup-sub{
+    padding:0 !important;
```
Comment (oyekanmiayo) [inline src/styles/login.css:200]: Codacy Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/Fields.jsx [hunk 26]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Field extends Component {
+  render() {
+    return (
+            <div className="form-group">
+                <label htmlFor={this.props.htmlFor} className='form-label'>{this.props.label}</label>
```
Comment (oyekanmiayo) [inline src/components/common/Fields.jsx:7]: Codacy Issue found: ['htmlFor' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 17 above)
Comment (oyekanmiayo) [inline src/styles/login.css:38]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 27]
```diff
@@ -0,0 +1,296 @@
...
+.landing>.login-wrapper{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+    height: 100%;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
```
Comment (oyekanmiayo) [inline src/styles/login.css:30]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 1 above)
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/InvestorsLoginSidebar.jsx [hunk 28]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import step1img from '../../images/steps (6).svg'
+
+class LoginSideBar extends Component {
+  render() {
+    return (
+            <div className='login-sidbar'>
+              <div className="sidebar-heading">
+                <h1>{this.props.heading} <span className='green-span'></span></h1>
```
Comment (oyekanmiayo) [inline src/components/common/InvestorsLoginSidebar.jsx:9]: Codacy Issue found: ['heading' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 9 above)
Comment (oyekanmiayo) [inline src/styles/login.css:45]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 19 above)
Comment (oyekanmiayo) [inline src/styles/login.css:15]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 11 above)
Comment (oyekanmiayo) [inline src/styles/login.css:11]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/InvestorsLoginSidebar.jsx [hunk 29]
```diff
@@ -0,0 +1,33 @@
...
+                          <p>
+                            <span className="blue-span">BuildForSDG</span> Connects you with the right SMEs that you can invest your money without fear of losing it all yet getting maximum returns.
+                          </p>
+                      </div>
+                      <div className="step-svg">
+                         <img src={step1img} alt="step1"/>
+                      </div>
+                  </div>
+              </div>
+
```
Comment (oyekanmiayo) [inline src/components/common/InvestorsLoginSidebar.jsx:28]: Codacy Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 22 above)
Comment (oyekanmiayo) [inline src/styles/login.css:33]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 30]
```diff
@@ -0,0 +1,296 @@
...
+}
+.landing>.login-wrapper{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+    height: 100%;
```
Comment (oyekanmiayo) [inline src/styles/login.css:29]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 31]
```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
+    margin:0;
```
Comment (oyekanmiayo) [inline src/styles/login.css:4]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 8 above)
Comment (oyekanmiayo) [inline src/styles/login.css:21]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/routes/beforeAuth.js (hunk 21 above)
Comment (oyekanmiayo) [inline src/components/routes/beforeAuth.js:5]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 3 above)
Comment (oyekanmiayo) [inline src/styles/login.css:3]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 8 above)
Comment (oyekanmiayo) [inline src/styles/login.css:21]: Codacy Issue found: [Expected single space before ">" (selector-combinator-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css (hunk 1 above)
Comment (oyekanmiayo) [inline src/styles/login.css:32]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/components/common/LoginSidebar.jsx [hunk 32]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import step1img from '../../images/bgsvg.svg'
```
Comment (oyekanmiayo) [inline src/components/common/LoginSidebar.jsx:2]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

File: src/styles/login.css [hunk 33]
```diff
@@ -0,0 +1,296 @@
...
+    80%{
+        content: 'Mature';
+    }
+    100%{
+        content: 'Connect';
+    }
+}
+.signup-sub{
+    padding:0 !important;
+    padding-bottom: 0 !important;
```
Comment (oyekanmiayo) [inline src/styles/login.css:201]: Codacy Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)

**Pull request closed without merging**